source/Government.cpp

```diff
@@ -277,6 +286,26 @@ void Government::Load(const DataNode &node)
 
 
 
+// Load a set of reputation titles and return them as a map.
+map<double, string> Government::LoadReputationTitles(const DataNode &node)
+{
+	map<double, string> result;
+	if(!node.HasChildren())
+		return result;
+	for(const DataNode &child : node)
+	{
+		if(child.Size() != 2)
```

Comment:
Arent we usually more generous with parsing?
```suggestion
		if(child.Size() >= 2)
```

Reply:
Allowing malformed lines just allows the possibility for mistakes.

Reply:
Agreed but thats how it happens for the rest of the game's parsing?